### PR TITLE
Integrate HANA test into upstream code

### PR DESCRIPTION
**Pull request by Amrysliu**
For now, we use `CASEDIR` to run HANA project on our test code. We would like to commit the related modification to upstream, then we could remove the setting `CASEDIR`.

Verification run: http://openqa.qa2.suse.asia/tests/60130#step/full_run/24

### CODE REVIEW

**Comment by JoyceNa** [comment]
LGTM

**Comment by jerrytt** [comment]
LGTM

File: tests/kernel_performance/install_qatestset.pm [hunk 1]
```diff
@@ -65,6 +80,7 @@ sub setup_environment {
         );
         assert_script_run("cat /root/qaset/qaset-setup.log");
     }
+
```
Comment (Dawei-Pang) [inline tests/kernel_performance/install_qatestset.pm:83]: please remove empty line, thanks!

**Comment by Dawei-Pang** [comment]
LGTM, thanks!

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**